.storybook/decorators/ComponentPreview.tsx

```diff
@@ -17,21 +18,17 @@ const darkModeStyle = css`
   background-color: ${palette.black};
 `;
 
-const decorator: Decorator = (Story, options) => {
-  const { darkMode, baseFontSize } = options.args as {
-    darkMode?: boolean;
-    // TODO: Use token
-    baseFontSize?: 14 | 16;
-  };
+const decorator: Decorator = (StoryFn: StoryFn, context: StoryContext<any>) => {
+  const { darkMode, baseFontSize } = context.args;
   return (
-    <LeafyGreenProvider
-      darkMode={darkMode as boolean}
-      baseFontSize={baseFontSize}
-    >
-      <div className={cx(rootStyle, { [darkModeStyle]: darkMode as boolean })}>
-        <div>
-          <Story darkMode={darkMode} {...options} />
-        </div>
+    <LeafyGreenProvider darkMode={darkMode} baseFontSize={baseFontSize}>
+      <div
+        className={cx({
+          [rootStyle]: context.name !== GENERATED_STORY_NAME,
```

Comment:
what's this doing?

Reply:
We only want to apply the story wrapper styles for non-generated stories